src/Illuminate/Console/Scheduling/ScheduleListCommand.php

```diff
@@ -109,7 +112,9 @@ public function handle(Schedule $schedule)
                 ? $nextDueDate->format('Y-m-d H:i:s P')
                 : $nextDueDate->diffForHumans();
 
-            $hasMutex = $event->mutex->exists($event) ? 'Has Mutex › ' : '';
+            $hasMutex = $event instanceof CallbackEvent
+                ? (is_string($event->description) && $event->mutex->exists($event) ? 'Has Mutex › ' : '')
+                : ($event->mutex->exists($event) ? 'Has Mutex › ' : '');
```

Comment:
`CallbackEvent` depends on unique `$event->description` to generate mutex cache key and it is optional.

Reply:
I tackled this differently by using an empty string fallback in the `mutexName` method of `CallbackEvent`.